test/system/tests/createak.sh

```diff
@@ -54,6 +56,13 @@ cleanup
 
 tpm2_createek -Q -H 0x8101000b -g rsa -p ek.pub
 
-tpm2_createak -Q -E 0x8101000b  -k 0x8101000c -g rsa -D sha256 -s rsassa -p ak.pub  -n ak.name
+tpm2_createak -Q -E 0x8101000b -k 0x8101000c -g rsa -D sha256 -s rsassa -p ak.pub -n ak.name
+
+# Find a vacant persistent handle
+tpm2_createak -E 0x8101000b -k - -g rsa -D sha256 -s rsassa -p ak.pub -n ak.name > ak.out
+phandle=`grep "ak-persistent-handle: " ak.out | awk '{print $2}'`
```

Comment:
Can you use something like yaml_get_kv() from helpers.sh here to get the value versus awk

This will take care of verifying via yaml.load() as well as fetching the value.

Reply:
here and throughout the changes

Reply:
Will do. The only place that causes a problem is in _getmanufec.sh_ which does a `tpm2_listpersistent | grep -q $phandle`. The problem is that the YAML conversion in yaml_get_kv converts the hex handle to a scalar, so we end up passing grepping for an int value.

There are two ways to resolve this:
1. quote the hex values we output so that they are always treated as a string
2. use a different YAML loader in yaml_get_kv function which doesn't do the conversion

Any thoughts?

Reply:
The third and simplest option is of course to drop the line `tpm2_listpersistent | grep -q $phandle`

Reply:
I went with the option of dropping that line from the test.